tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -1186,7 +1186,7 @@ public:
          compensation = (t - sum) - y;
          sum = t;
       }
-   *fResultSum = sum;
+   *fResultSum += sum;
```

Comment:
could we simplify this by removing the local `sum` variable and directly accumulate into `fResultSum` in the loop above?